pkg/plugins/plugin_test.go

```diff
@@ -59,10 +60,14 @@ func testActive(t *testing.T, p *Plugin) {
 }
 
 func TestGet(t *testing.T) {
-	t.Parallel()
+	// TODO: t.Parallel()
+	// TestPluginWithNoManifest also registers fruitPlugin
```

Comment:
😞 good catch on the package-level var.

Perhaps we can "fix" this if we pick different names for these? Did you look if there's a specific reason they were using the same name, or was that just "convenience"?

Reply:
Yeah I tried to change the names to be different, but it didn't work (was causing the test to hang and timeout) and I decided to just fix the flakiness for now.